client/src/CodeClaimPage/QRHashForm.tsx

```diff
@@ -69,6 +74,26 @@ const QRHashForm: React.FC<{error: boolean, loading: boolean, checkClaim: (hash:
         </Formik>
       </div>
       <ClaimFooterMessage />
+
+      <ReactModal isOpen={isModalOpen} shouldFocusAfterRender={true}>
```

Comment:
Please prefer implicit prop boolean over explicit

```suggestion
      <ReactModal isOpen={isModalOpen} shouldFocusAfterRender>
```

As a side note if you find an ESLint rule for this to add would be great 👍

Reply:
<img width="369" alt="Screen Shot 2020-07-28 at 15 50 35" src="https://user-images.githubusercontent.com/8381041/88708481-15823180-d0ea-11ea-825d-7f49c1b95003.png">

done